app/services/doc_auth_router.rb

```diff
@@ -217,7 +217,7 @@ def self.doc_auth_vendor(discriminator: nil)
   end
 
   def self.randomize?(target_percent, discriminator)
-    max_sha = (16 ** 64) - 1
+    max_sha = (16**64) - 1
```

Comment:
This was a bit surprising, but appears to be the expected default ([source](https://github.com/rubocop/rubocop/blob/54f4c82286c725a28a57216158d05f409a94eb7c/spec/rubocop/cop/layout/space_around_operators_spec.rb#L205-L207)). It can be configured though, if we'd prefer ([docs](https://docs.rubocop.org/rubocop/cops_layout.html#configurable-attributes-44)).

Reply:
I'm gonna call an audible and reinstate the spaces. The inconsistency of the default behavior is pretty jarring to me.

[dd9cce3](https://github.com/18F/identity-idp/pull/6236/commits/dd9cce3a0e972dbcf29d44f2b620974c4290d247)